src/system/unlockables.ts

```diff
@@ -8,6 +10,16 @@ export enum Unlockables {
   EVIOLITE
 }
 
+export function isUnlocked(unlockable: Unlockables, gameData: GameData): boolean {
+  if (Overrides.UNLOCK_OVERRIDE.includes(unlockable)) {
+    return true;
+  }
+  if (Overrides.DISABLE_UNLOCK_OVERRIDE.includes(unlockable)) {
+    return false;
+  }
+  return !!gameData.unlocks[Unlockables.MINI_BLACK_HOLE];
```

Comment:
```suggestion
  return gameData.unlocks[unlockable];
```
As flx said, as is the function would default to `MINI_BLACK_HOLE` regardless of the `unlockable`. Moreover, because `unlocks` is already a key-value pair giving back a `boolean`, there's no need for the `!!`.